src/components/paneler/innhold/personaliaV2/epost.tsx

```diff
@@ -23,9 +23,17 @@ function Epost(props: { epost: OrNothing<PersonaliaEpost> }) {
 
 	return (
 		<Informasjonsbolk header="Epost" {...rest}>
-			<Normaltekst className="innrykk wrap-anywhere">
+			<Normaltekst className="innrykk wrap-anywhere display-flex">
 				{epostAdresse}
-				{epostAdresse && <Kopiknapp kopitekst={epostAdresse} type="epost" />}
+				{epostAdresse && (
+					<CopyToClipboard
+						copyText={epostAdresse}
+						popoverText="Kopiert"
```

Comment:
Nå er jo konteksten ganske tydelig men jeg ser ikke noe i veien for å repetere konteksten i slike tekster for å hjelpe brukeren i størst mulig grad. Feks:

```suggestion
						popoverText="E-postadresse ble kopiert"
```

Reply:
Jeg syns det i så fall holder med "E-postadresse kopiert" 🤔

Reply:
Kari svarte at det var greit å bare ha "kopiert" for nå i alle fall. Dette vil ikke bli plukket opp av skjermleser eller liknende så det er tydelig hvilken kontekst du står i, slik du nevner Sondre.